Refs #34380 -- Added FORMS_URLFIELD_ASSUME_HTTPS transitional setting. (#17538)
django/django: docs/releases/5.0.txt

* The ``ForeignObject.get_reverse_joining_columns()`` method is deprecated.

* The default scheme for ``forms.URLField`` will change from ``"http"`` to

>   ``"https"`` in Django 6.0. Set :setting:`EARLY_ADOPT_URLFIELD_ASSUME_HTTPS`
>   transitional setting to ``True`` if you want to early adopt assuming
>   ``"https"`` during the Django 5.x release cycle.

Reviewer: We can drop the word “early” here since we’re being explicit about the release cycle.

```suggestion
  ``"https"`` in Django 6.0. Set :setting:`EARLY_ADOPT_URLFIELD_ASSUME_HTTPS`
  transitional setting to ``True`` to opt into assuming ``"https"``
  during the Django 5.x release cycle.
```
